src/demo/ShareTooltip/ShareTooltip.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../components/variables.scss';
```

Comment:
`src/demo` folder is only for technical components. Put this under `__stories__` folder